README.md

```diff
@@ -159,10 +165,10 @@ value stored in Redis, you could do something like:
 
 ```ruby
 task resque:pool:setup do
-Resque::Pool.config_loader = lambda {|env|
-  worker_count = Redis.current.get("pool_workers_#{env}").to_i
-  {"queueA,queueB" => worker_count }
-}
+  Resque::Pool.config_loader = lambda do |env|
+    worker_count = Redis.current.get("pool_workers_#{env}").to_i
+    {"queueA,queueB" => worker_count }
+  end
```

Comment:
I've always preferred the Weirich Semantic Rule for blocks: https://github.com/bbatsov/ruby-style-guide/issues/162

But I'll admit I seem to be in the minority.  ;)